src/css/components/Icon.scss

```diff
@@ -17,3 +17,44 @@
   top: -1px; // 1
   fill: currentColor; // 2
 }
+
+// * Modifier for loader icon
+
+$Icon--loader-offset: 130;
+$Icon--loader-duration: 1.4s;
+
+.Icon--loader {
+  animation: rotator $Icon--loader-duration linear infinite;
+  fill: none;
+  stroke: currentColor;
+
+  circle {
+    stroke-dasharray: $Icon--loader-offset;
+    stroke-dashoffset: 0;
+    transform-origin: center;
+    animation: dash $Icon--loader-duration ease-in-out infinite
+  }
+}
+
+@keyframes rotator {
```

Comment:
I have one in the loading icons that does the exact same thing... might be good to consolidate these keyframe animations at some point

Reply:
Ah, yeah. That would be nice, like a new file in config or something.

Reply:
yeah